### PR TITLE
Update sponsors

### PR DESCRIPTION
**Pull request by dns2utf8**
We should merge this probably before: https://github.com/RustFestEU/blog.rustfest.eu/pull/24

### CODE REVIEW

**Review by Hoverbear — CHANGES_REQUESTED** [review]
Please add Parity and Thoughtworks

**Comment by Hoverbear** [comment]
tw_logo_black

**Comment by dns2utf8** [comment]
Added TW

File: _data/sponsors.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-- name: "bitfury"
-  logoName: bitfury.png
+- name: "Parity"
+  logoName: parity_logo_transparent_white.png
   featured: true
-  link: "http://bitfury.com/"
+  link: "https://parity.io/"
   group: sponsor
-  desc: "The Bitfury Group develops and delivers both the software and the hardware solutions necessary for businesses, governments, organizations and individuals to securely move an asset across the Blockchain. The expertise of The Bitfury Group ensures successful, easy, fast, secure and cost-effective connectivity to the Blockchain."
+  desc: "The Parity Group develops and delivers both the software and the hardware solutions necessary for businesses, governments, organizations and individuals to securely move an asset across the Blockchain. The expertise of The Parity Group ensures successful, easy, fast, secure and cost-effective connectivity to the Blockchain."
```
Comment (Hoverbear) [inline _data/sponsors.yml]: This is not okay. :-P

 Please just make it blank until they give us something.

**Review by Hoverbear — APPROVED** [review]

**Pull request merged**